lib/src/react16_suggestors/react_dom_render_migrator.dart

```diff
@@ -65,8 +71,19 @@ class ReactDomRenderMigrator extends GeneralizingAstVisitor
       return ancestor is FunctionDeclaration;
     });
 
+    void addOverReactPatch(int offset){
```

Comment:
`offset` ins't used in the function, so to get it to work in a single line:

```suggestion
    void addOverReactPatch(){
```

Or (I personally like better) update to use the offset passed in as a parameter.